corehq/ex-submodules/casexml/apps/case/mock/case_block.py

```diff
@@ -96,16 +96,16 @@ def _to_json(self):
         if self.index:
             result['index'] = {}
             for name in self.index.keys():
-                self.case_type = self.index[name].case_type
-                self.case_id = self.index[name].case_id
+                case_type = self.index[name].case_type
+                case_id = self.index[name].case_id
```

Comment:
Pretty sure this was a bug. It was likely never noticed before since `CaseBlock`s are rarely referenced after being posted.